packages/core/src/components/label/label.tsx

```diff
@@ -18,16 +21,29 @@ export class Label {
     ['danger', 'Gevaar']
   ]);
 
+  @Prop()
+  removable?: boolean;
+
+  @State()
+  hovered?: boolean;
+
+  @Event()
+  actionClick!: EventEmitter<MouseEvent>;
+
   render() {
     const status = this.status && Label.statusMap.get(this.status);
 
     return (
-      <span class={clsx('dso-label', { [`dso-label-${this.status}`]: this.status } )}>
+      <span class={clsx('dso-label', { [`dso-label-${this.status}`]: this.status, 'dso-label-small': this.small, 'dso-label-hover': this.hovered })}>
         {status && (
           <span class="sr-only">{status}: </span>
         )}
         <slot></slot>
-        <slot name="action"></slot>
+        {this.removable && (
+          <button type="button" onClick={e => this.actionClick.emit(e)} title="Verwijder" slot="action" onMouseEnter={() => this.hovered = true} onMouseLeave={() => this.hovered = false}>
```

Comment:
* Waarom is `slot="action"` nodig?
* Hoe zit het met `focus` stijling?
![image](https://user-images.githubusercontent.com/9696261/120485275-370c1380-c3b4-11eb-8a96-ae3b54bff3f2.png)

Reply:
Ik heb een aanpassing moeten doen in `icon.scss`, namelijk een `line-height: 0` zetten op `.icon-container`. De line-height zorgde er voor dat deze container naar buiten geduwt werd.

Reply:
Ook heb ik er voor gezorgd dat de tekst nu doorstreept wordt wanneer de knop focus heeft